src/app/dataset-block/metadata-block/components/yaml-view-section/yaml-view-section.component.ts

```diff
@@ -12,8 +13,8 @@ import { SupportedEvents } from "../event-details/supported.events";
 })
 export class YamlViewSectionComponent extends BaseComponent implements OnInit {
     public block: MetadataBlockFragment;
```

Comment:
Similarly, let's move the `block` to `BlockService`

Reply:
Moved block to BlockService